prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'NadiaKim';
+    console.log(fullName);
+    var number0fpets = '1';
+    console.log(number0fpets);
```

Comment:
Hi Nadia, be sure to leave off the quotes from numbers so that the browser doesn't interpret them as strings